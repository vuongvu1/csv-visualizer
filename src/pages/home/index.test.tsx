import React from "react";
import { render, cleanup } from "@testing-library/react";
import Home from "./index";

describe("Home page", () => {
  beforeEach(cleanup);

  test("renders learn react link", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
