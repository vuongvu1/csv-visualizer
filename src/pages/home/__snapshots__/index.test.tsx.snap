// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders learn react link 1`] = `
<div>
  <div>
    This is home page
  </div>
</div>
`;
